include/modem/location.h

```diff
@@ -64,34 +64,47 @@ enum location_event_id {
 	/**
 	 * Application has indicated that getting location has been completed,
 	 * the result is not known, and the Location library does not need to care about it.
+	 *
 	 * This event can occur only if @kconfig{CONFIG_LOCATION_SERVICE_EXTERNAL} is set.
 	 */
 	LOCATION_EVT_RESULT_UNKNOWN,
 	/**
-	 * GNSS is requesting A-GNSS data. Application should obtain the data and send it to
-	 * location_agnss_data_process().
+	 * GNSS is requesting A-GNSS data.
+	 *
+	 * Application should obtain the data and send it to location_agnss_data_process().
 	 */
 	LOCATION_EVT_GNSS_ASSISTANCE_REQUEST,
 	/**
-	 * GNSS is requesting P-GPS data. Application should obtain the data and send it to
-	 * location_pgps_data_process().
+	 * GNSS is requesting P-GPS data.
+	 *
+	 * Application should obtain the data and send it to location_pgps_data_process().
 	 */
 	LOCATION_EVT_GNSS_PREDICTION_REQUEST,
 	/**
 	 * Cloud location request with neighbor cell and/or Wi-Fi access point information
-	 * is available. The application should send the information to cloud services and
-	 * then call location_cloud_location_ext_result_set().
+	 * is available.
+	 *
+	 * The application should send the information to cloud services and then call
+	 * location_cloud_location_ext_result_set().
 	 */
-	LOCATION_EVT_CLOUD_LOCATION_EXT_REQUEST
+	LOCATION_EVT_CLOUD_LOCATION_EXT_REQUEST,
+	/**
+	 * Location rewquest has been started.
```

Comment:
This was already in my earlier suggestion and I thought you might miss it. 😄 
```suggestion
	 * Location request has been started.
```